__tests__/app.test.js

```diff
@@ -31,7 +31,28 @@ describe("app",()=>{
         test("status 404 - path not found", ()=>{
             return request(app).get('/bad-pathway')
             .expect(404)
-
-            })
+            .then(({body})=>{
+                expect(body.msg).toEqual("Path not found")})
         })
-    })
\ No newline at end of file
+    })
+    
+    describe("GET /api/articles/:article_id",()=>{
```

Comment:
We could add further testing here for when this isn't the data we're looking for. Hint, what KIND of data should it be?